### DISCUSSION TITLE
AaC Logo

### DISCUSSION THREAD

**Discussion by jondavid-black** [DevOps-MBSE/AaC#687] [General]
I've been thinking it would be nice to have some sort of visual representation for AaC.  In the early days I played around with some crappy boxes and letters stuff that would never pass muster, so I glad it didn't go in and get any momentum.  Now that we have the power of AI I thought it would be fun to see what a much more talented digital artist would come up with.

Here's the prompt I gave to Midjourney (with some variation along the way):  product logo for a code based model-based system engineering tool called architecture as code that uses simple geometric line art resembling a 3D engineering blueprint with shades of blue on a black background with as few lines and surfaces as possible

![aac_logo_1](https://github.com/DevOps-MBSE/AaC/assets/36022341/318cca57-ee9b-48c8-9972-7a2b5106fbc8)
![aac_logo_2](https://github.com/DevOps-MBSE/AaC/assets/36022341/ae062dbf-f8d3-4902-8418-71e4854e3fec)
![aac_logo_3](https://github.com/DevOps-MBSE/AaC/assets/36022341/103642a3-26a6-4869-888c-fa5484ff5ba9)
![aac_logo_4](https://github.com/DevOps-MBSE/AaC/assets/36022341/7513a522-b896-4e8f-8770-9c046e0e96d4)

I kinda like the 3rd one best initially.  I wouldn't have come up with using a tree, but the metaphor of MBSE branching into and connecting everything across a project / program is very appealing to me.  I also like the idea of architecture having an organic nature to it.

So what do you think?  Got a favorite?  Want to try some different prompts?  Let me know.

**Comment by Coffee2Bits** [comment]
I'm a fan of the 3rd one as well. I can't think of an alternative logo for AaC -- it's hard for me to visualize it as something other than the interconnected blocks you've got with the 1st and 4th pictures. That being said, I think the tree is a more recognizable and better logo design.

**Comment by lizzcondrey** [comment]
I like the 3rd best also, but I'm a huge fan of the tree of life/knowledge imagery so not surprised that's the one I was pulled to.
